experimental/packages/otlp-transformer/src/logs/internal.ts

```diff
@@ -79,17 +79,20 @@ function logRecordsToResourceLogs(
   encoder: Encoder
 ): IResourceLogs[] {
   const resourceMap = createResourceMap(logRecords);
-  return Array.from(resourceMap, ([resource, ismMap]) => ({
-    resource: createResource(resource),
-    scopeLogs: Array.from(ismMap, ([, scopeLogs]) => {
-      return {
-        scope: createInstrumentationScope(scopeLogs[0].instrumentationScope),
-        logRecords: scopeLogs.map(log => toLogRecord(log, encoder)),
-        schemaUrl: scopeLogs[0].instrumentationScope.schemaUrl,
-      };
-    }),
-    schemaUrl: undefined,
-  }));
+  return Array.from(resourceMap, ([resource, ismMap]) => {
+    const processedResource = createResource(resource);
+    return {
+      resource: processedResource,
+      scopeLogs: Array.from(ismMap, ([, scopeLogs]) => {
+        return {
+          scope: createInstrumentationScope(scopeLogs[0].instrumentationScope),
```

Comment:
Could this throw if `scopeLogs` is empty?

Reply:
Possibly, but the only change here in this PR is creating `processedResource` and passing on the schemaUrl

Reply:
Going to resolve this as I think it doesn't affect the changes proposed in this PR.